op-scim-bridge/Makefile

```diff
@@ -48,5 +48,8 @@ app/build:: .build/check/docker .build/op-scim/deployer .build/op-scim/deps
 .build/op-scim/deps:
 	./push-dep-image-tags.sh
 
+.build/op-scim:
```

Comment:
Added this build step as the directory not existing was causing errors on my freshly-checked-out repo.